litgpt/utils.py

```diff
@@ -305,7 +305,7 @@ def get_default_supported_precision(training: bool) -> str:
 
     if MPSAccelerator.is_available() or (torch.cuda.is_available() and not torch.cuda.is_bf16_supported()):
         return "16-mixed" if training else "16-true"
-    return "bf16-mixed" if training else "bf16-true"
```

Comment:
Ah yes, I maybe should have discussed this. The issue here is that it will be very very slow and resource intensive by default. But I am happy to not do this change.